### PR TITLE
Permit BeatmapInfoWedge to display Unicode artist and title

### PR DESCRIPTION
**Pull request by ColdVolcano**

### CODE REVIEW

**Comment by peppy** [comment]
We really need to start thinking about centralising this logic inside BeatmapMetadata itself

File: osu.Game/Screens/Select/BeatmapInfoWedge.cs [hunk 1]
```diff
@@ -102,6 +105,9 @@ private void load()
                 CacheDrawnFrameBuffer = true;
                 RelativeSizeAxes = Axes.Both;
 
+                titleBinding = localisation.GetUnicodePreference(metadata.TitleUnicode, metadata.Title);
```
Comment (peppy) [inline osu.Game/Screens/Select/BeatmapInfoWedge.cs]: you should probably be binding to both of these separately. what happens if the title changes but the artist doesn't?

**Review by peppy — CHANGES_REQUESTED** [review]

File: osu.Game/Screens/Select/BeatmapInfoWedge.cs (hunk 1 above)
Comment (ColdVolcano) [reply osu.Game/Screens/Select/BeatmapInfoWedge.cs]: then this needs to be fixed in various places

Comment (peppy) [reply osu.Game/Screens/Select/BeatmapInfoWedge.cs]: quite possibly, yeah. I think we should probably have the artist/title stuff handled inside BeatmapMetadata (ie. give it a ctor localisationManager argument) then have BeatmapMetadata expose a `Bindable<string> DisplayTitle` which it culminates the artist/title/events into

**Review by peppy — APPROVED** [review]

**Comment by peppy** [comment]
gonna merge this for now, it's clean enough

**Pull request merged**